games/bzflag/Portfile

```diff
@@ -4,10 +4,10 @@ PortSystem          1.0
 PortGroup           app 1.0
 
 name                bzflag
-version             2.4.12
+version             2.4.14
 categories          games
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @allejo
```

Comment:
You need to list both your email address and your github handle here, in braces. See other ports for examples, or the guide. Also, I would recommend that, unless there's a good reason not to, you also set `openmaintainer` to indicate you don't mind minor changes being made by MacPorts team members.

Reply:
Added my email and `openmaintainer` as well